package.json

```diff
@@ -67,8 +69,10 @@
     "browserify": "^17.0.0",
     "budo": "^11.6.4",
     "cross-env": "^7.0.3",
+    "cypress": "^7.0.1",
```

Comment:
Looks like the latest release is 7.1.0 now. Could you update the dependency here, as well as the container used in CI to `cycpress/included:7.1.0`?